CMakeLists.txt

```diff
@@ -163,3 +163,15 @@ endif()
 
 # Add docs
 add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/docs)
+
+# Format our code
+if(CLANG_FORMAT_EXE)
+    file(GLOB_RECURSE FILES_TO_FORMAT "${CMAKE_CURRENT_SOURCE_DIR}/src/*.cpp"
```

Comment:
don't we have variables listing the filenames for all the libCellML source, header files, tests, etc. ? And if so, shouldn't we use those rather than globbing all files, which might include non-libCellML files?

Reply:
True, but they are "buried" in sub CMakeLists.txt files. So, I guess we have two choices: 1) keep things as they or 2) call clang-format in all sub CMakeLists.txt files, which is a bit of a pain and something that we might forget to do if we were ever going to add a sub CMakeLists.txt file.

Happy to go with whatever the majority decides though.

Reply:
We could use `PARENT_SCOPE` to pass the full list of the files to format up to this CMakeLists.txt file.

Reply:
I am in favour of the `PARENT_SCOPE` approach. Ok, will look into it sometime this week.